schemas/remote_configuration_v2_template.schema.json

```
@@ -0,0 +1,41 @@
+{
+  "$id": "remote_configuration_v2_template",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "additionalProperties": false,
+  "description": "Config template files stored as object which contain list of conditional gathering rules and list of container logs",
+  "example": {
+    "conditional_gathering_rules": [
+      "conditional_gathering_rules/rule1.json",
+      "conditional_gathering_rules/rule2.json",
+      "conditional_gathering_rules/subdir/rule3.json"
+    ],
+    "container_logs": [
+      "container_log_requests/manual/kube_controller_manager_logs_gatherer.json"
```

It would be nice if the example demonstrated that the items are glob patterns, e.g. like this

```suggestion
      "container_log_requests/ccx-rules-ocp/*.json",
      "container_log_requests/manual/kube_controller_manager_logs_gatherer.json"
```

FYI: The subdirectories will be used to track sources of the requests, and allow teams to manage their requests without interfering with other teams. Unlike conditional gathering, the Rapid Recommendations feature is designed to handle duplicate requests efficiently.

---

Code updated as requested